MapboxDirections/MBRouteStep.swift

```diff
@@ -643,10 +643,10 @@ open class RouteStep: NSObject, NSSecureCoding {
             maneuverDirection = nil
         }
         
-        guard let roadSideDescription = decoder.decodeObject(of: NSString.self, forKey: "roadSide") as String?, let roadSide = RoadSide(description: roadSideDescription) else {
+        guard let drivingSideDescription = decoder.decodeObject(of: NSString.self, forKey: "drivingSide") as String?, let drivingSide = DrivingSide(description: drivingSideDescription) else {
```

Comment:
Let’s default this value to `right` if it’s unspecified, so that preexisting objects don’t suddenly fail to decode.